src/CadViewer.tsx

```diff
@@ -1,4 +1,4 @@
-import type { AnySoupElement } from "@tscircuit/soup"
+import type { AnySoupElement } from "circuit-json"
```

Comment:
```suggestion
import type { AnyCircuitElement } from "circuit-json"
```

Use this at all the other places as well